apps/src/templates/codeReviewGroups/AssignedStudentsPanel.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import i18n from '@cdo/locale';
+import FontAwesome from '@cdo/apps/templates/FontAwesome';
+import JavalabButton from '@cdo/apps/javalab/JavalabButton';
+import StudentGroup from './StudentGroup';
+import color from '@cdo/apps/util/color';
+import {HEADER_STYLE, BUTTON_STYLE} from './UnassignedStudentsPanel';
+
+export default function AssignedStudentsPanel({groups, onCreateGroupClick}) {
```

Comment:
Question for @madelynkasula - do we have a preference for using arrow functions for functional components? (regardless though, nice functional component!)

Reply:
i think preference is to use the style ben's using here. there's a couple of reasons i personally prefer this style:

1. this is how functional components are presented in the [react docs](https://reactjs.org/docs/components-and-props.html#function-and-class-components)
2. AFAIK it's more complicated to `export default` a named constant, and we want our components to always be named. example of what i mean:

```javascript
// exporting unnamed constant
export default (props) => {};

// exporting named constant as default
const MyComponent = props => {};
export default MyComponent;

// invalid js
export default const MyComponent = props => {};
```